repo-issues/lib/template.html

```diff
@@ -0,0 +1,47 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<link rel="stylesheet" href="{{{__dirname}}}/style.css">
+
+<div class="container">
+    {{#repos}}
+    <div class="cell">
+        <div class="header">
+            <span class="title">
+                <img class="icon" 
+                    alt="{{language}}" 
+                    src="https://cdn.rawgit.com/konpa/devicon/master/icons/{{language}}/{{language}}-plain.svg"
+                    onerror="this.style.display='none'"/>
+                {{name}}
+            </span>
+            {{^show_details}}
+            <div class="issue">
```

Comment:
if we find our templates getting big we can extract partials out with moustache.

Like 

```
{{^show_details}}
    {{> details-partial}}
{{/show_details}}
```

```
- index.html
- details.partial
```


 although the base dashboard plugin will need updating to support them but is probably worth doing as our plugins get more complicated

Reply:
Nice one, I was wondering how to do it as this template is quite messy already.
I'd say lets add support for it on the base plugin and come back to this plugin (and maybe others) after